deps/chakrashim/src/v8v8.cc

```diff
@@ -142,18 +139,22 @@ void V8::SetFlagsFromCommandLine(int *argc, char **argv, bool remove_flags) {
         argv[i] = nullptr;
       }
     } else if (equals("--trace-debug-json", arg) ||
-      equals("--trace_debug_json", arg)) {
+               equals("--trace_debug_json", arg)) {
       g_trace_debug_json = true;
       if (remove_flags) {
         argv[i] = nullptr;
       }
-    } else if (remove_flags &&
-               (startsWith(
-                 arg, "--debug")  // Ignore some flags to reduce unit test noise
-                || startsWith(arg, "--harmony")
-                || startsWith(arg, "--stack-size=")
-                || startsWith(arg, "--nolazy"))) {
-      argv[i] = nullptr;
+    } else if (startsWith(arg, "--debug") ||
+               startsWith(arg, "--harmony") ||
+               startsWith(arg, "--stack-size=") ||
+               startsWith(arg, "--nolazy") ||
+               startsWith(arg, "--max-old-space-size=")) {
+      // Ignore some flags to reduce unit test noise
+      fprintf(stderr, "Warning: Ignored engine flag: %s\n", arg);
```

Comment:
This doesn't seem to bother any of the unit tests, but I'll have to check that VS Code doesn't get upset about it (I doubt it).  I'd say we either have a warning for all of these flags or none of them.